[2023-04-26T04:17:55.079+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_available manual__2023-04-26T04:17:53.136488+00:00 [queued]>
[2023-04-26T04:17:55.088+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_available manual__2023-04-26T04:17:53.136488+00:00 [queued]>
[2023-04-26T04:17:55.089+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-04-26T04:17:55.089+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-04-26T04:17:55.090+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-04-26T04:17:55.099+0000] {taskinstance.py:1383} INFO - Executing <Task(HttpSensor): is_api_available> on 2023-04-26 04:17:53.136488+00:00
[2023-04-26T04:17:55.106+0000] {standard_task_runner.py:55} INFO - Started process 21400 to run task
[2023-04-26T04:17:55.111+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'is_api_available', 'manual__2023-04-26T04:17:53.136488+00:00', '--job-id', '135', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpsiegabiv']
[2023-04-26T04:17:55.118+0000] {standard_task_runner.py:83} INFO - Job 135: Subtask is_api_available
[2023-04-26T04:17:55.218+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.is_api_available manual__2023-04-26T04:17:53.136488+00:00 [running]> on host bebb68623a17
[2023-04-26T04:17:55.289+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=is_api_available
AIRFLOW_CTX_EXECUTION_DATE=2023-04-26T04:17:53.136488+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-04-26T04:17:53.136488+00:00
[2023-04-26T04:17:55.291+0000] {http.py:120} INFO - Poking: api/
[2023-04-26T04:17:55.303+0000] {base.py:71} INFO - Using connection ID 'user_api' for task execution.
[2023-04-26T04:17:55.309+0000] {http.py:148} INFO - Sending 'GET' to url: https://randomuser.me/api/
[2023-04-26T04:17:56.118+0000] {base.py:213} INFO - Success criteria met. Exiting.
[2023-04-26T04:17:56.147+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=is_api_available, execution_date=20230426T041753, start_date=20230426T041755, end_date=20230426T041756
[2023-04-26T04:17:56.195+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-04-26T04:17:56.232+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
